### PR TITLE
Name change to AKS Landing Zone Accelerator

### PR DESCRIPTION
**Pull request by mosabami**
Change the name Enterprise Scale for AKS  to AKS Landing Zone Accelerator

### CODE REVIEW

**Comment by PRMerger5** [comment]
@mosabami : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by mosabami** [comment]
please review and approve @BrianBlanchard @JanetCThomas

**Comment by PRMerger20** [comment]
@mosabami : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger5** [comment]
@mosabami : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by mosabami** [comment]
#sign-off

**Review by TomGeske — APPROVED** [review]
LGTM

**Comment by PRMerger5** [comment]
@JanetCThomas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Comment by mosabami** [comment]
> @mosabami - Please verify this link. - line 23 [Start with Cloud Adoption Framework AKS landing zone accelerator](../../ready/enterprise-scale/index.md) It points to a generic landing zone topic. the AKS landing zone topic is here - /scenarios/aks/ready.md Also, always use lower case for landing zone accelerators. Thanks

**Comment by mosabami** [comment]
> @mosabami - Please verify this link. - line 23 [Start with Cloud Adoption Framework AKS landing zone accelerator](../../ready/enterprise-scale/index.md) It points to a generic landing zone topic. the AKS landing zone topic is here - /scenarios/aks/ready.md Also, always use lower case for landing zone accelerators. Thanks

@JanetCThomas that is the correct link, we want it to go to the Azure Landing Zones home page. we want them to deploy that first before AKS. Do you want me to make the changes to lower case now? or has that been taken care of and that was for future submissions?

File: docs/scenarios/aks/enterprise-scale-landing-zone.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+AKS landing zone accelerator assumes that a platform foundation that takes care of the shared services (network, security, identity, and governance) required to effectively construct and operationalize a landing zone has been successfully implemented. This isn't mandatory when implementing AKS landing zone accelerator, but it does handle much of the security and management required to safely manage your cloud environment. If you already have one, you can skip this step. For more information on this, review these articles:
 
-- [Start with Cloud Adoption Framework enterprise-scale landing zones](../../ready/enterprise-scale/index.md)
-- [Implement Cloud Adoption Framework enterprise-scale landing zones in Azure](../../ready/enterprise-scale/implementation.md)
+- [Start with Cloud Adoption Framework AKS landing zone accelerator](../../ready/enterprise-scale/index.md)
```
Comment (dominicallen) [inline docs/scenarios/aks/enterprise-scale-landing-zone.md:23]: I think this line should be removed (as it duplicates the line below)

File: docs/scenarios/aks/enterprise-scale-landing-zone.md [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+- [Implement Cloud Adoption Framework Azure landing zone](../../ready/enterprise-scale/implementation.md)
```
Comment (dominicallen) [inline docs/scenarios/aks/enterprise-scale-landing-zone.md:24]: I think this should point here: /ready/landing-zone/

**Comment by PRMerger13** [comment]
@mosabami : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@mosabami : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger4** [comment]
@JanetCThomas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by JanetCThomas** [comment]
#sign-off  This has been reviewed by the content team and can be merged.

**Comment by PRMerger18** [comment]
@JanetCThomas: I'm sorry - only the author of this article, @BrianBlanchard, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by GitHubber17** [comment]
Thank you.

**Pull request merged**